src/app/components/reportes/encabezadoReportes.html

```diff
@@ -28,8 +28,21 @@
                 </plex-wrapper>
             </plex-grid>
         </form>
-        <reporteC2 *ngIf="showReporteC2" [diagnosticos]="diagnosticos" (select)="diagnostico = $event"></reporteC2>
-        <cantidadConsultaXPrestacion *ngIf="showCantidadConsultaXPrestacion" [diagnosticos]="diagnosticos">
+        <plex-title *ngIf="!inicio" titulo="Listado de reportes" size="sm"></plex-title>
+        <div *ngIf="!inicio && estaVacio" justify="center" class="h-50">
```

Comment:
Sugiero evaluar directamente el length de los resultados, y prescindir del flag `estaVacio`. De esta manera escribimos menos lineas de código y nos evitamos tener que esta actualizando el flag en el component cada vez que actualizamos los resultados
```suggestion
        <div *ngIf="!inicio && diagnosticos.length" justify="center" class="h-50">
```

Reply:
Corregido!